README.md

```diff
@@ -74,29 +74,33 @@ err := flow.Run(ctx, input, &output)
 ## Architecture Overview
 
 ```mermaid
+---
+config:
+  layout: elk
+---
 flowchart TB
-    subgraph subGraph0["Streaming Pipeline<br>io.Pipe<br>Goroutines"]
+ subgraph subGraph0["&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Streaming Pipeline<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;with io.Pipe &<br>&nbsp;&nbsp;&nbsp;Goroutines"]
```

Comment:
not sure npsp works here
Removed those entities and keeping <br> for line breaks seems reasonable